docs/dev-guides/PERSONAL-ENVIRONMENTS.md

```diff
@@ -0,0 +1,523 @@
+    ├── global.yaml
+    ├── hydra
+    ├── icdemo
+    ├── identityconcentrator
+    ├── istioconfig
+    ├── istioconfig.yaml
+    ├── leonardo
+    ├── ontology
+    ├── poc
+    ├── poc.yaml
+    ├── rawls
+    ├── sam
+    ├── workspacemanager
+    └── workspacemanager.yaml
+```
+For this PR we'll edit all of
+* the helmfile itself, to make it aware of our personal enviornment
+* a personal.yaml file defining our environment's enabled services at a  high level.
+* several values files defining parameters for the individual services in the environment
+
+### Helmfile
+The main helmfile itself gets a simple two-line entry for this project pointing to other files with
+properties to read (including the one above):
+```yaml
+  jcarlton:
+    values:
+      - environments/personal.yaml
+      - environments/personal/jcarlton.yaml
+```
+
+### Personal yaml file
+This file describes the Terra apps that are required for the personal environment. In my case those were
+resource buffer, janitor, and workspace manager. For the file `terra-helmfile/environments/personal/jcarlton.yaml`:
+```yaml
+# Environment overrides for jcarlton environment
+releases:
+  crljanitor:
+    enabled: true
+  workspacemanager:
+    enabled: true
+  buffer:
+    enabled: true
+```
+If any environments are disabled, their resources are not generated by Terraform.
+
+The main personal.yaml is nearly empty (and should not be edited). It appears just to be a lightweight
+bit of plumbing so the whole repo isn't just one massive file.
+<details>
+
+<summary>personal.yaml</summary>
+
+```yaml
+# Common settings for private developer environments
+base: 'personal'
+
+# All private dev environments go to same cluster
+defaultCluster: terra-integration
+
+# Enable services below, eg:
+# releases:
+#   cromwell:
+#     enabled: true
+```
+
+</details>
+
+Finally, each service to be included has its own `values` override file for our environment:
+For `terra/values/workspacemanager/personal/jcarlton.yaml`, you should verify that [vault](https://www.vaultproject.io/) has secret
+stored at the path described. In my case this had already been taken care of.
+
+```yaml
+vault:
+  pathPrefix: secret/dsde/terra/kernel/integration/jcarlton
+
+ingress:
+  staticIpName: terra-integration-jcarlton-workspace-ip
+```
+
+For `terra/values/crljanitor/personal/jcarlton.yaml`, you need the IP address from the `atlantis apply` call in the previous PR.
+```yaml
+vault:
+  pathPrefix: secret/dsde/terra/kernel/integration/jcarlton
+  configPathPrefix: config/terraform/terra/integration/jcarlton
+serviceIP: 35.223.31.120
+```
+
+For `buffer`, there are additional properties:
+```yaml
+vault:
+  pathPrefix: secret/dsde/terra/kernel/integration/jcarlton
+
+ingress:
+  staticIpName: terra-integration-jcarlton-buffer-ip
+
+buffer:
+  pool:
+    configPath: config/toolsalpha
+
+proxy:
+  whitelist:
+    email: "buffer-client-jcarlton@terra-kernel-k8s.iam.gserviceaccount.com"
+```
+## Creating the namespace
```

Comment:
Prior headings at this level were numbered "Step 1" and "Step 2". Should this be "Step 3: Creating the namespace"?

Reply:
good point